azurerm/internal/services/databricks/resource_arm_databricks_workspace.go

```diff
@@ -59,6 +59,7 @@ func resourceArmDatabricksWorkspace() *schema.Resource {
 				ValidateFunc: validation.StringInSlice([]string{
 					"standard",
 					"premium",
+					"trial",
```

Comment:
could we also [update the documentation to reflect this new value](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/databricks_workspace.html.markdown#L43)?

Reply:
I updated it and submitted this PR 

https://github.com/terraform-providers/terraform-provider-azurerm/pull/5666